repo_health/index/static/client/src/main.module.js

```diff
@@ -25,20 +27,52 @@ export const main = angular.module('repo-health', [
     uiRouter,
     ngMaterial,
     components,
-    resources
+    resources,
+    'chart.js'
   ])
   .component('main', mainComponent)
-  .config(($locationProvider, $urlRouterProvider, $mdThemingProvider) => {
+  .service('StatsService', StatsService)
+  .config(($locationProvider, $urlRouterProvider, $mdThemingProvider, $stateProvider) => {
     'ngInject';
-    
     $locationProvider.hashPrefix('');
     $urlRouterProvider.otherwise('/search');
+    $stateProvider.state('repo-health', {
+        url: '/repo-health/:owner__login/:name',
+        resolve: {
+            statsUrls: (RepoDetailsService, $stateParams, $state) => {
+                return RepoDetailsService.getStatsUrls($stateParams)
+                    .then(resp => {return resp;}, () => { return {error:true}});
```

Comment:
spaces between he brackets makes for nice readability `{ return { error: true } }`.